testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java

```diff
@@ -29,7 +29,9 @@
 import io.trino.spi.connector.ConnectorFactory;
 import io.trino.spi.connector.ConnectorViewDefinition;
```

Comment:
Do you have test for `CREATE TABLE LIKE` where we should see that columns have nice source columns set.

Reply:
In case of `LIKE` we don't capture the data right ? We just reuse the name and type.